packages/blade/src/components/Card/__tests__/CardInteractive.web.test.tsx

```diff
@@ -0,0 +1,225 @@
+import userEvent from '@testing-library/user-event';
+import React from 'react';
+import { CardBody, Card } from '../';
+import renderWithTheme from '~utils/testing/renderWithTheme.web';
+import { Box } from '~components/Box';
+import { VisuallyHidden } from '~components/VisuallyHidden/VisuallyHidden';
+import { Text } from '~components/Typography';
+
+describe('<Card />', () => {
+  it('should render a interactive Card', () => {
+    const { container } = renderWithTheme(
+      <Card onClick={() => {}} scaleOnHover isSelected={true}>
+        <CardBody>Plain Card</CardBody>
+      </Card>,
+    );
+    expect(container).toMatchSnapshot();
+  });
+
+  it('should render a clickable Card', () => {
+    const { container } = renderWithTheme(
+      <Card onClick={() => {}}>
+        <CardBody>Plain Card</CardBody>
+      </Card>,
+    );
+    expect(container).toMatchSnapshot();
+  });
+
+  it('should render a linkable Card', () => {
+    const { container } = renderWithTheme(
+      <Card href="https://google.com">
+        <CardBody>Plain Card</CardBody>
+      </Card>,
+    );
+    expect(container).toMatchSnapshot();
+  });
+
+  it('should render a linkable Card with onClick', () => {
+    const { container } = renderWithTheme(
+      <Card onClick={() => {}} href="https://google.com">
+        <CardBody>Plain Card</CardBody>
+      </Card>,
+    );
+    expect(container).toMatchSnapshot();
+  });
+
+  it('should click on the card', async () => {
+    const user = userEvent.setup();
+    const onClick = jest.fn();
+    const { getByRole } = renderWithTheme(
+      <Card onClick={onClick} accessibilityLabel="My Card">
+        <CardBody>Plain Card</CardBody>
+      </Card>,
+    );
+    await user.click(getByRole('button', { name: /My Card/i }));
+    expect(onClick).toHaveBeenCalled();
+  });
+
+  it('should behave properly when buttons are present', async () => {
+    const user = userEvent.setup();
+    const rootOnClick = jest.fn();
+    const childOnClick = jest.fn();
+    const { getByRole } = renderWithTheme(
+      <Card onClick={rootOnClick} accessibilityLabel="My Card">
+        <CardBody>
+          <button onClick={childOnClick}>Child Button</button>
+        </CardBody>
+      </Card>,
+    );
+
+    await user.tab();
+    expect(getByRole('button', { name: /My Card/i })).toHaveFocus();
+    await user.click(getByRole('button', { name: /My Card/i }));
+    expect(rootOnClick).toHaveBeenCalled();
+
+    await user.tab();
+    expect(getByRole('button', { name: /Child Button/i })).toHaveFocus();
+    await user.click(getByRole('button', { name: /Child Button/i }));
+    expect(childOnClick).toHaveBeenCalled();
+  });
+
+  it('should have href on card', () => {
+    const { getByRole } = renderWithTheme(
+      <Card href="https://google.com" accessibilityLabel="My Card">
+        <CardBody>Plain Card</CardBody>
+      </Card>,
+    );
+    expect(getByRole('link', { name: /My Card/i })).toHaveAttribute('href', 'https://google.com');
```

Comment:
how do we put and check `rel` attribs like we do for links?

Reply:
Why do we need to check for rel? rel isn't passed in.

Reply:
Oh i see you mean the defaultRel, added after adding the rel prop.